contracts/metagov/VeBalHelper.sol

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import {VeBalDelegatorPCVDeposit} from "./VeBalDelegatorPCVDeposit.sol";
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+import {SafeERC20, IERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {TransparentUpgradeableProxy, ProxyAdmin} from "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+
+/// @title Helper contract for veBAL OTC
+/// @author eswak
+contract VeBalHelper is Ownable {
```

Comment:
To make sure I follow, we can't just transfer the `veBAL`. Instead we need to create this wrapper contract which the OTC buyer will use to effectively assume ownership - and it exposes the various needed methods like `lock()`, `setGaugeController()` etc which are decoupled from Tribe's access permissions

Reply:
💯 correct